### PR TITLE
Describe how variants are represented in the webhook

### PR DESCRIPTION
**Pull request by aleks-tpom6oh**

### CODE REVIEW

File: pages/publishing/publishing/webhook-publishing.mdx [hunk 1]
```diff
@@ -59,8 +59,14 @@ Json representation of your tracking plan, following the [Json Schema](https://j
 
 ##### Event Model
 
+> Event variants in the JSON payload:
+> Each event variant is represented by a standalone event in the JSON payload. It will have the same `name`, as the base event.
+> You can distinguish an event, that is generated from a variant, from a base/standalone event, by checking for a `.` symbol in the id. 
+> The event id of events that are generated from variants have the `${baseEventId}.${variantId}` structure.
```
Comment (logason) [inline pages/publishing/publishing/webhook-publishing.mdx:65]: I think we're using the `<Callout />` component for layouting callouts like this one nicelye

Comment (logason) [reply pages/publishing/publishing/webhook-publishing.mdx:65]: Suggest moving this below the table, since it's more of a detail

**Review by logason — APPROVED** [review]
Two suggestions but no blockers, LGTM!

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…acking-plan/properties**](https://docs-1g8tau9cf.avo-dev.app/data-design/avo-tracking-plan/properties) | [**…ing/webhook-publishing**](https://docs-1g8tau9cf.avo-dev.app/publishing/publishing/webhook-publishing) |
|:-:|:-:|
|<a href="https://docs-1g8tau9cf.avo-dev.app/data-design/avo-tracking-plan/properties">Screenshot of /data-design/avo-tracking-plan/properties</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/4PK11WL_mPII6HxAMObAp_1Jf7vC9V4N8MhEj_wnQcxBf_6UliH-en7PZyNPrF6zSEzrwWntoOYq6XXx6cK7KQ.png">(view full size)</a> |<a href="https://docs-1g8tau9cf.avo-dev.app/publishing/publishing/webhook-publishing">Screenshot of /publishing/publishing/webhook-publishing</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/F-BXakvZMf4OGycZuzl72btBbTxcLSlxE5Kn_l0CrLi1zi1JFw0IY9tMtpxRWHZHtfO3Gx_vI-uAC5sRrhh5jQ.png">(view full size)</a> |







Commit 813a2ceb3df060def4c87c02c69275d9547a4baa (https://docs-1g8tau9cf.avo-dev.app).

**Pull request merged**